pkg/volume/azure_dd/attacher.go

```diff
@@ -268,7 +268,7 @@ func (d *azureDiskDetacher) Detach(diskURI string, nodeName types.NodeName) erro
 		return fmt.Errorf("invalid disk to detach: %q", diskURI)
 	}
 
-	_, err := d.cloud.InstanceID(context.TODO(), nodeName)
+	instanceid, err := d.cloud.InstanceID(context.TODO(), nodeName)
```

Comment:
context.TODO() seems like a mistake here? is there a context we can attach to?

Reply:
@brendanburns it's related to https://github.com/kubernetes/kubernetes/pull/59287, there is some future work item for `context.TODO()`